config/db.config.js

```diff
@@ -0,0 +1,47 @@
+export const HOST = 'localhost';
+export const USER = 'postgres';
+export const PASSWORD = 'Barbaris179$';
+export const DB = 'postgres';
+export const dialect = 'postgres';
```

Comment:
Alos take a look at migrations/seeds concepts
https://sequelize.org/docs/v6/other-topics/migrations/#creating-the-first-seed